pytest_slack/plugin.py

```diff
@@ -126,5 +134,7 @@ def pytest_terminal_summary(terminalreporter, exitstatus, config):
                "attachments": [results_pattern],
                "icon_emoji": emoji if icon is None else None,
                "icon_url": icon}
-
-    requests.post(slack_hook, data=json.dumps(payload), timeout=timeout)
+    if disable_ssl_verify is False:
```

Comment:
If we make ssl_verify option which is default = True
then make this just one line
requests.post(slack_hook, data=json.dumps(payload), timeout=timeout, verify=ssl_verify)